packages/preset-mini/src/_rules/color.ts

```diff
@@ -23,8 +23,13 @@ export const bgColors: Rule[] = [
       return { 'background-size': h.bracketOfLength(d)!.split(' ').map(e => h.fraction.auto.px.cssvar(e) ?? e).join(' ') }
     if ((isSize(d) || bgPositionRE.test(d)) && h.bracketOfPosition(d) != null)
       return { 'background-position': h.bracketOfPosition(d)!.split(' ').map(e => h.position.fraction.auto.px.cssvar(e) ?? e).join(' ') }
-    if (bgGradientRE.test(d) || bgImageRE.test(d))
-      return { 'background-image': h.bracket(d) }
+    if (bgGradientRE.test(d) || bgImageRE.test(d)) {
+      const s = h.bracket(d)
+      if (s) {
+        const url = s.startsWith('http') ? `url(${s})` : h.cssvar(s)
```

Comment:
The `url` seem to unnecessary for `bgGradientRE` case. 
How about it only for `bgImageRE` case?

Reply:
`bgGradientRE` and `bgImageRE` both produce `background-image` css, we can handle them together.